documentation/v0_9_0.md

```diff
@@ -0,0 +1,368 @@
+---
```

Comment:
Can we drop the raw release notes I created in GitHub Releases to the bottom of this?

Reply:
We could, but do we really want a copy of them on the webpage?